collated/main/vicisapotato.md

```diff
@@ -0,0 +1,416 @@
+# vicisapotato
+###### \java\seedu\address\commons\events\ui\JumpToTabRequestEvent.java
+``` java
+public class JumpToTabRequestEvent extends BaseEvent {
+
+    public final int targetIndex;
+
+    public JumpToTabRequestEvent(Index targetIndex) {
+        this.targetIndex = targetIndex.getZeroBased();
+    }
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName();
+    }
+
+}
+```
+###### \java\seedu\address\commons\events\ui\ShowParcelListEvent.java
+``` java
+public class ShowParcelListEvent extends BaseEvent {
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName();
+    }
+
+}
+```
+###### \java\seedu\address\logic\commands\ListCommand.java
+``` java
+        EventsCenter.getInstance().post(new ShowParcelListEvent());
+```
+###### \java\seedu\address\logic\commands\TabCommand.java
+``` java
+public class TabCommand extends Command {
+    public static final String COMMAND_WORD = "tab";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": changed the tab displayed in the parcel list panel "
+            + "Parameters: INDEX (must be a positive integer) "
+            + "Example: " + COMMAND_WORD + " 1 ";
+
+    public static final String MESSAGE_SELECT_TAB_SUCCESS = "Selected Tab: %1$s";
+    public static final String MESSAGE_INVALID_TAB_INDEX = "Invalid Tab Number";
+
+    public static final int NUM_TAB = 2;
+
+    private final Index targetIndex;
+
+    /**
+     * @param targetIndex of the tab in the TabPane to switch to
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)